source/fundamentals/versioned-api.txt

```diff
@@ -0,0 +1,123 @@
+=============
+Versioned API
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The Versioned API feature requires MongoDB Server 5.0 or later. You
+   should only use the Versioned API feature if you are connecting to
+   MongoDB servers that are upgraded to a version that supports this feature.
+
+This guide shows you how to specify the **Versioned API** when connecting to
+a MongoDB instance or replica set. You can use the Versioned API feature to
+force the server to run operations with consistent behavior as specified by
+an **API version**. An API version defines expected behavior of operations
```

Comment:
Good catch, I agree that the current wording is confusing. I think the suggestion might not be the change I'm looking for though since it emphasizes behavior specified by the API version instead of the operations for that version of the API.
My sentence might also be inaccurate from reading the C# documentation on Versioned API [here](https://mongodb.github.io/mongo-csharp-driver/2.13/reference/driver/versioned_api/):

> During communication with a server, clients with a declared API version will force the server to behave in a manner compatible with the API version. eclaring an API version on a client can be used to ensure consistent responses from a server, providing long term API stability for an application.

If that documentation is correct, I should emphasize compatible behavior for operations rather than consistent, and consistent responses from the server.